### ISSUE TITLE
Generating OnCalendar incorrectly?

### ISSUE THREAD

**Issue by wilbowma** [systemd-cron/systemd-cron-next#34]
I just switched from systemd-cron to this, and non of my crontabs are working correctly. It seems the generator is leaving the "@" symbol when generating OnCalendar lines, which does not seem to be valid in a .timer file.
E.g., one of the generated timer files reads:

```
[Timer]
Unit=cron-21738b156814949b0736d48293195d4a.service
Persistent=true
OnCalendar=@weekly
```

While I think it should read
```
[Timer]
Unit=cron-21738b156814949b0736d48293195d4a.service
Persistent=true
OnCalendar=weekly
```

This is happening with all my crontabs, each of which uses either hourly, daily, weekly, or monthly.

**Comment by a-detiste** [comment]
Timers are broken in systemd 229

**Comment by a-detiste** [comment]
Ha, sorry for the noise, this is something else... for @kstep

**Comment by kstep** [comment]
Thank you for the report. I will look into it this evening and let you know of results.

**Comment by kstep** [comment]
The bug should be fixed in 1.0.0-rc7.
